### PR TITLE
fix issue #1131

### PR DESCRIPTION
**Pull request by kz33**
Fixing issue：#1131

### CODE REVIEW

**Comment by kz33** [comment]
@googlebot I signed it!

File: charts/spark-operator-chart/values.yaml [hunk 1]
```diff
@@ -136,3 +136,6 @@ leaderElection:
 istio:
   # -- When using `istio`, spark jobs need to run without a sidecar to properly terminate
   enabled: false
+
+# labelSelectorFilter -- A comma-separated list of key=value, or key labels to filter resources during watch and list based on the specified labels.
+labelSelectorFilter: ""
```
Comment (liyinan926) [inline charts/spark-operator-chart/values.yaml:141]: Add a newline at the end of the file.

**Review by liyinan926 — COMMENTED** [review]
Please also update README.md for the new config option.

**Comment by liyinan926** [comment]
Please update the chart version.

**Comment by kz33** [comment]
hi,@liyinan926,Can you help me see the reason why ci hasn't passed it? Thanks so much !

**Comment by liyinan926** [comment]
Can you do some dummy commit to trigger the Travis build?

**Review by liyinan926 — APPROVED** [review]
LGTM

**Pull request merged**